src/catalogue/docker/docker-compose-zipkin.yml

```diff
@@ -0,0 +1,52 @@
+#
+# Copyright (c) 2019, Oracle and/or its affiliates. All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+# 
+
+version: '2'
+
+services:
+    catalogue:
+        build:
+            context: ..
+            dockerfile: ./Dockerfile
+        image: mushop/catalogue
+        hostname: catalogue
+        restart: always
+        cap_drop:
+            - all
+        cap_add:
+            - NET_BIND_SERVICE
+        read_only: true
+        volumes:
+            - Wallet_Creds:/usr/lib/oracle/19.3/client64/lib/network/admin/
+        environment:
+            - reschedule=on-node-failure
+            - OADB_USER=catalogue_user
+            - OADB_PW=default_Password1
+            - OADB_SERVICE=mcatalogue_tp
+            - ZIPKIN=http://zipkin:9411/api/v1/spans
+        ports:
```

Comment:
Thinking we don’t actually want this file, do we?

Reply:
It's the only use for [Zipkin](https://landscape.cncf.io/selected=zipkin) and [OpenTracing](https://landscape.cncf.io/selected=open-tracing) right now. This will keep evolving for the whole solution.

Reply:
Well then maybe pull out pwds?

Reply:
The pwd is a placeholder, also present on the docker-compose.yml that do the build locally. we can change to something more placeholdy, such as `change_it`

Reply:
Also is part of the .sql file.

Reply:
The name of the repo is xxx-cloudnative. How to use .env using the zip stack and ORM? The upcoming alternative deployment using K8s and OSB will generate all password automatically.

Reply:
Yes I am on the same page with these remarks, I just was not aware those features were being actively folded into this repository and the upcoming release.

As for `.env` in the basic/ORM deployment, some values are already set dynamically on init. `OADB_SERVICE` is dynamic today, but `OADB_PW` is hard-coded and should be converted to a generated value used when creating the schema. Is there a reason for hesitation with using `docker-compose` + `.env` now, when it's already possible?

Reply:
ORM uses the same sql file. If implement to set the password dynamically the TF script, then we can change all.

Reply:
I realize that... I'm just saying that it seems like a bad practice to _create_ tech debt when there's no reason to do so in this file.

Reply:
docker-compose.yml is already there. This just includes Zipkin for who want to do tracing/test with the service. The password placeholder is already present on the main use of deployment. By addressing this, need to change the TF script to not use the password placeholder for the limited-read-only DB user.

Reply:
I would remove this zipkin file completely as it is not used anywhere at all. We are using source code control, there is history and we can always bring the zipkin compose file back if neededd.

Reply:
> docker-compose.yml is already there. This just includes Zipkin for who want to do tracing/test with the service. The password placeholder is already present on the main use of deployment. By addressing this, need to change the TF script to not use the password placeholder for the limited-read-only DB user.

Should be removed from the docker-compose.yml too. `mcatalogue_tp` is a completely different service anyway... changing these to use environment variables now doesn't mean any changes necessary to the TF script. You can leave the `CREATE USER...` statement as-is until TF can be updated to generate this. If these files are present, I'm only suggesting to move these values to an ignored `.env` file for the present time.

Reply:
Changing the entries to variables, matching the service usage. Not including new info for `.env`. It's up to the developer if wants to use EXPORT Variable or .env files.

Reply:
I think we can add this file back in when required.

Reply:
@peterj that all good by you?